src/admin-views/settings/emails/message-templates.php

```diff
@@ -13,8 +13,12 @@
 if ( empty( $templates ) ) {
 	return;
 }
-// @todo Update template HTML.
+
 ?>
-<h3>
-	This space is reserved for the message template list.
-</h3>
+<div class="tec_tickets-emails-template-list">
+	<?
+	foreach ( $templates as $email_template ) {
+		$this->template( 'message-templates/list-item', [ 'template' => $email_template ] );
+	}
```

Comment:
We tend to use `:` instead of `{` and  and `endforeach;` on templates.

```suggestion
	foreach ( $templates as $email_template ) : ?>
		<?php $this->template( 'message-templates/list-item', [ 'template' => $email_template ] ); ?>
	<?php endforeach; ?>
```